modules/storages/lib/api/v3/storage_files/storage_files_api.rb

```diff
@@ -54,16 +54,44 @@ def raise_error(error)
               files_query
                 .call(params[:parent])
                 .map do |files|
-                  API::V3::StorageFiles::StorageFileCollectionRepresenter.new(
-                    files,
-                    self_link: api_v3_paths.storage_files(@storage.id),
-                    current_user:
-                  )
-                end
-                  .match(
-                    on_success: ->(representer) { representer },
-                    on_failure: ->(error) { raise_error(error) }
-                  )
+                API::V3::StorageFiles::StorageFileCollectionRepresenter.new(
+                  files,
+                  self_link: api_v3_paths.storage_files(@storage.id),
+                  current_user:
+                )
+              end
+                .match(
+                  on_success: ->(representer) { representer },
+                  on_failure: ->(error) { raise_error(error) }
+                )
+            },
+            on_failure: ->(error) { raise_error(error) }
+          )
+      end
+
+      # RequestBody:
+      # {
+      #   "fileName": "ape.png",
+      #   "parent": "/Pictures"
+      # }
```

Comment:
the request body is only passed through, do we need to have a inline comment here?

Reply:
No, I will remove it in one of the upcoming WPs.

Reply:
I'd like to keep it now for reference.